### DISCUSSION TITLE
Provide access to column-oriented query results

### DISCUSSION THREAD

**Discussion by cocoa-xu** [ClickHouse/clickhouse-go#1318] [Feature requests]
Hi, thanks for this great library for ClickHouse! 

I was wondering if it's possible to access query results by column? Currently `Conn.Query` only exposed row-oriented results via `Rows`. It'd be much more easier to work with other column-major libraries such as arrow if column-oriented results are available.

**Comment by jkaflik** [comment]
hi @cocoa-xu [ch-go](https://github.com/ClickHouse/ch-go) fits your needs.

**Reply by cocoa-xu** [reply]
Thanks @jkaflik! I'll take a look at it.
